tidb-cloud/tidb-cloud-security-password-authentication.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Password Authentication
+summary: Learn how to manage TiDB Cloud console passwords and how to enable multi-factor authentication (MFA).
+---
+
+# Password Authentication
+
+This document describes how to manage TiDB Cloud console passwords and how to enable multi-factor authentication (MFA). It is applicable to users who [sign up](https://tidbcloud.com/free-trial) for TiDB Cloud with emails and passwords.
+
+## Sign up
+
+If you want to manage your TiDB Cloud passwords using the TiDB Cloud console, you can sign up for a TiDB Cloud account using your email and password. 
```

Comment:
I checked with @lulukelu about this. This doc provides the signup section also as a placeholder to add more relevant content in the future.  If user want to use TiDB Cloud to manage their password, they must sign up with email and password instead of using SSO.